tests/test_events.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Test that the work package service consumes and processes events properly."""
+
+from hexkit.providers.akafka.testutils import (  # noqa: F401 # pylint: disable=unused-import
+    kafka_fixture,
+)
+from hexkit.providers.mongodb.testutils import (  # noqa: F401 # pylint: disable=unused-import
+    mongodb_fixture,
+)
+from pytest import mark, raises
+
+from .fixtures import fixture_container  # noqa: F401 # pylint: disable=unused-import
+from .fixtures.datasets import DATASET
+
+
+@mark.asyncio
+async def test_dataset_registration(container):
```

Comment:
I feel it makes it difficult to find the fixture when it has a different name than the actual fixture function. Added to the refactoring list. Maybe we can discuss this at some point.

Maybe also consider adding a type hints to understand what the fixture returns.

Reply:
Yes, was a bit sloppy here. But fixtures are already magical enough, so adding type hints is a good idea. Done in 2d477cd.